source/reference/config/triggers.txt

```diff
@@ -241,6 +319,34 @@ trigger configuration files:
        | ``object``
      - .. include:: /includes/trigger-project-expression.rst
 
+   * - | ``event_processors.config.account_id``
+       | ``string``
+     - An AWS account ID. For more information on how to find the account ID,
+       see step 1 (Setup the MongoDB Partner Event Source) of the procedure on the 
```

Comment:
```suggestion
       refer to the 
```
i think they'd be able to find it in the procedure

Reply:
It's a long page, I'd rather they find it easily. Edited